bolt/lib/Profile/BoltAddressTranslation.cpp

```diff
@@ -492,6 +486,10 @@ uint64_t BoltAddressTranslation::translate(uint64_t FuncAddress,
 
   const uint32_t Val = KeyVal->second >> 1; // dropping BRANCHENTRY bit
   if (IsBranchSrc) {
+    // Branch entry is found in BAT
+    if (KeyVal->first == Offset && KeyVal->second & BRANCHENTRY)
```

Comment:
What would be the case where we need to check for the second condition (`KeyVal->second & BRANCHENTRY`)? Can we drop it?